### PR TITLE
SCCP-356-Base-Sepolia

### PR DESCRIPTION
**Pull request by kaleb-keny**

### CODE REVIEW

File: tomls/omnibus-base-sepolia-andromeda/perps/markets/arkm.toml [hunk 1]
```diff
@@ -22,6 +22,7 @@ target = ["perpsFactory.PerpsMarketProxy"]
 fromCall.func = "owner"
 func = "createMarket"
 args = ["<%= settings.arkm_perps_market_id %>", "Arkham", "ARKM"]
+args = ["<%= settings.arkm_perps_market_id %>", "0", "ARKM"]
```
Comment (noisekit) [inline tomls/omnibus-base-sepolia-andromeda/perps/markets/arkm.toml:25]: Oops

File: tomls/omnibus-base-sepolia-andromeda/perps/markets/eos.toml [hunk 2]
```diff
@@ -21,7 +21,6 @@ eos_perps_endorsed_liquidator = "0x11233749514Ab8d00C0A5873DF7428b3db70030f"
 target = ["perpsFactory.PerpsMarketProxy"]
 fromCall.func = "owner"
 func = "createMarket"
-args = ["<%= settings.eos_perps_market_id %>", "Eos", "EOS"]
```
Comment (noisekit) [inline tomls/omnibus-base-sepolia-andromeda/perps/markets/eos.toml:24]: Another one

File: tomls/omnibus-base-sepolia-andromeda/perps/markets/ftm.toml [hunk 3]
```diff
@@ -21,7 +21,6 @@ ftm_perps_endorsed_liquidator = "0x11233749514Ab8d00C0A5873DF7428b3db70030f"
 target = ["perpsFactory.PerpsMarketProxy"]
 fromCall.func = "owner"
 func = "createMarket"
-args = ["<%= settings.ftm_perps_market_id %>", "Fantom", "FTM"]
```
Comment (noisekit) [inline tomls/omnibus-base-sepolia-andromeda/perps/markets/ftm.toml:24]: More

File: tomls/omnibus-base-sepolia-andromeda/perps/markets/arkm.toml (hunk 1 above)
Comment (kaleb-keny) [reply tomls/omnibus-base-sepolia-andromeda/perps/markets/arkm.toml]: yeah was still a work in progress 😅  now fixed

File: tomls/omnibus-base-sepolia-andromeda/perps/markets/ftm.toml (hunk 3 above)
Comment (kaleb-keny) [reply tomls/omnibus-base-sepolia-andromeda/perps/markets/ftm.toml]: this one too

**Review by noisekit — APPROVED** [review]

**Comment by noisekit** [comment]
No unexpected changes in this preview https://github.com/Synthetixio/Synthetix-Gitbook-v3/pull/481/files

**Pull request merged**